### PR TITLE
Move consultations to universal layout

### PR DESCRIPTION
**Pull request by vanitabarrett**
**Before:**
screen shot 2018-01-02 at 11 08 04

**After:**
screen shot 2017-12-22 at 11 40 38

**Links to Test:**
- [Open Consultation](https://government-frontend-pr-650.herokuapp.com/government/consultations/keeping-children-safe-in-education-proposed-revisions)
- [Closed Consultation](https://government-frontend-pr-650.herokuapp.com/government/consultations/offensive-and-dangerous-weapons-new-legislation)
- [Consultation Outcome](https://government-frontend-pr-650.herokuapp.com/government/consultations/short-inspections-of-good-schools-maintained-schools-and-academies)
- [Consultation with History Notice](https://government-frontend-pr-650.herokuapp.com/government/consultations/science-advisory-council-sac-and-the-advisory-committee-on-releases-to-the-environment-acre-triennial-review-2014)
- [Consultation with Feedback](https://government-frontend-pr-650.herokuapp.com/government/consultations/uk-smi-v-37-chlamydia-trachomatis-infection-testing-by-nucleic-acid-amplification-tests-naat)
- [Consultation with Ways to Respond](https://government-frontend-pr-650.herokuapp.com/government/consultations/review-of-the-smoke-and-carbon-monoxide-alarm-regulations-2015)

Review app component guide:
https://government-frontend-pr-650.herokuapp.com/component-guide

### CODE REVIEW

File: app/views/content_items/consultation.html.erb [hunk 1]
```diff
@@ -1,175 +1,193 @@
...
+      <% end %>
+    <% end %>
 
-<% if @content_item.final_outcome? %>
-  </section>
-<% end %>
+      <% consultation_desc = capture do %>
+        <%= @content_item.description %>
+        <% if @content_item.held_on_another_website? %>
+          <br/><br/>
```
Comment (maxgds) [inline app/views/content_items/consultation.html.erb]: This looks a bit hacky - could it be a separate paragraph instead? ie wrap each thing in p tags ? Or is the spacing specifically trying to look different from a paragraph?

File: app/views/content_items/consultation.html.erb [hunk 2]
```diff
@@ -1,175 +1,193 @@
...
+          <% end %>
+
+          <% if @content_item.email %>
+            <h3>Email to:</h3>
+            <p><%= mail_to @content_item.email, @content_item.email %></p>
+          <% end %>
+
+          <% if @content_item.postal_address %>
+            <h3>Write to:</h3>
+            <div class="contact">
```
Comment (maxgds) [inline app/views/content_items/consultation.html.erb]: Looks like a potentially leaky class name. Would it make sense to update to BEM throughout? Or remove classes that aren't being used for styling?

File: app/views/content_items/consultation.html.erb [hunk 3]
```diff
@@ -1,175 +1,193 @@
...
+      <%= render 'components/notice', title: 'We are analysing your feedback', description_text: content_item_final_outcome, margin_bottom: 0 %>
+
+    <% elsif @content_item.final_outcome? %>
+      <%= render 'components/notice', title: 'This consultation has concluded' %>
+
+      <% if @content_item.final_outcome_documents? %>
         <%= render 'components/heading', text: "Download the full outcome", id: "final-outcome-documents-title" %>
-      </div>
-      <div class="column-two-thirds" aria-labelledby="final-outcome-documents-title">
+        <div aria-labelledby="final-outcome-documents-title">
```
Comment (fofr) [inline app/views/content_items/consultation.html.erb]: As the heading and content are now inline, we could probably remove this div and the aria attribute.

File: app/views/content_items/consultation.html.erb [hunk 4]
```diff
@@ -1,175 +1,193 @@
...
           <% end %>
-        <% end %>
 
-        <% if @content_item.response_form? %>
-          <p>
-            Complete a <%= link_to 'response form', @content_item.attachment_url %> and
-            <% if @content_item.email && @content_item.postal_address %>either<% end %>
-          </p>
-        <% end %>
+          <% if @content_item.response_form? %>
```
Comment (fofr) [inline app/views/content_items/consultation.html.erb]: You've got two `response_forms`

File: app/views/content_items/consultation.html.erb (hunk 1 above)
Comment (fofr) [reply app/views/content_items/consultation.html.erb]: This isn't really a change, it's a whitespace edit. If you review with `?w=1` in the Github URL you'll see these hacks were there originally. I think it's ok to leave as-is for now.

File: app/views/content_items/consultation.html.erb (hunk 2 above)
Comment (fofr) [reply app/views/content_items/consultation.html.erb]: Same as above, this isn't really a change, it's a whitespace edit. I think it's ok to leave as-is for now.

**Pull request merged**